### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by sofa-on-the-sofa**

### CODE REVIEW

File: homework_1/js/hw_1.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+console.log (Math.min(15, 11, 16, 12, 51, 12, 13, 51));
+
+//Работа с Math.random: Получить случайное число и округлить его до двух цифр после запятой
+console.log (Math.random().toFixed(2));
+
+//Работа с Math.random: Получить случайное целое число от 0 до X. X - любое произвольное число
+console.log (Math.floor(Math.random() * (24+1)));
+
+//Проверить результат вычисления 0.6 + 0.7 - как привести к нормальному виду (1.3)?
+console.log (0.6 + 0.7 == 1.3);
```
Comment (moroz112) [inline homework_1/js/hw_1.js:52]: there is no need space here. Just console.log()

File: homework_1/js/hw_1.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+let string = 'some test string';
+
+// Получить первую и последнюю буквы строки
+console.log (string.length);
+console.log (string[0]);
+console.log (string[15]);
+
+// Сделать первую и последнюю буквы в верхнем регистре
+let myString = string[0].toUpperCase() + string.slice(1,15) + string[15].toUpperCase();
```
Comment (moroz112) [inline homework_1/js/hw_1.js:11]: Could use let myString = string[0].toUpperCase() + string.slice(1, string.length -1) + string[string.length -1].toUpperCase();

**Review by moroz112 — APPROVED** [review]
Really good job. It was pleasure reviewing it. You can merge your PR and continue training with learn.javascript.ru

File: homework_1/js/hw_1.js (hunk 2 above)
Comment (sofa-on-the-sofa) [reply homework_1/js/hw_1.js:11]: Got it. Thank you for review.

**Pull request merged**